staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -609,6 +613,10 @@ type VGPUDisplayOptions struct {
 	RamFB *FeatureState `json:"ramFB,omitempty"`
 }
 
+type Panic struct {
+	Model string `json:"model,omitempty"`
```

Comment:
Can you make this a typed string with some constants defining the supported values? For example:

https://github.com/kubevirt/kubevirt/blob/09a45ff11041456e54f5c2a596c5975ed57e840c/staging/src/kubevirt.io/api/core/v1/schema.go#L30-L38